runtime/gc_base/GCObjectEvents.cpp

```diff
@@ -31,18 +31,19 @@
 #include "mmomrhook_internal.h"
 
 #include "EnvironmentBase.hpp"
+#include "ForwardedHeader.hpp"
 #include "HeapMap.hpp"
 #include "MemorySpace.hpp"
 #include "MemorySubSpace.hpp"
 #include "MemorySubSpaceSemiSpace.hpp"
 #include "MemorySubSpaceRegionIterator.hpp"
+#include "ObjectAccessBarrier.hpp"
```

Comment:
I guess, this was somehow brought from SFH, but I wonder why it was needed  - does not seem to be obvious. Perhaps it was indirectly including something else that is more specific, in which case it should be used directly instead...

Reply:
Yes, but it seems we only need `Heap.hpp` so I'll substitute the above over `#include "Heap.hpp"`

Reply:
Sounds good.

While trying to understand why it was needed I observed also being included in a few other files without a clear reason, including one that you touched: CopyForwardScheme.cpp. Please, check why/if it was needed there, too.